suite-native/device/src/hooks/useHandleDeviceConnection.ts

```diff
@@ -44,6 +48,14 @@ export const useHandleDeviceConnection = () => {
     const { isBiometricsOverlayVisible } = useIsBiometricsOverlayVisible();
     const isDeviceUsingPassphrase = useSelector(selectIsDeviceUsingPassphrase);
     const isFirmwareInstallationRunning = useSelector(selectIsFirmwareInstallationRunning);
+
+    const isFirmwareCheckFailed = useSelector(selectHasFirmwareAuthenticityCheckHardFailed);
```

Comment:
please always keep the name of the const same as the selector name
```suggestion
    const hasFirmwareAuthenticityCheckHardFailed = useSelector(selectHasFirmwareAuthenticityCheckHardFailed);
```

Reply:
Done in 4f186034